_search-plugins/knn/knn-index.md

```diff
@@ -118,12 +118,13 @@ Lucene HNSW implementation ignores `ef_search`  and dynamically sets it to the v
 You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. faiss has
 several encoder types, but the plugin currently only supports *flat* and *pq* encoding.
 
-An example method definition that specifies an encoder may look something like this:
+An example method definition that specifies an encoder (method `hnsw` using `pq` encoder) may look something like this:
 
 ```json
 "method": {
   "name":"hnsw",
   "engine":"faiss",
+  "space_type": "l2",
```

Comment:
Can we remove code_size parameter. Due to some reasons, it has to be 8: https://github.com/opensearch-project/k-NN/blame/main/src/main/java/org/opensearch/knn/index/util/Faiss.java#L102. I think it is worth mentioning.

Reply:
I have highlighted that statement by including it in note. I think it is better to include that parameter in example or else users might be confused looking at the note.